SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -570,7 +570,13 @@ void USpatialLatencyTracer::WriteKeyFrameToTrace(const TraceSpan* Trace, const F
 
 FString USpatialLatencyTracer::FormatMessage(const FString& Message) const
 {
-	return FString::Printf(TEXT("%s(%s) : %s"), *MessagePrefix, *WorkerId.Left(18), *Message);
+	if (TraceMetadata.IsEmpty()) {
```

Comment:
Have you considered making the metadata apply only to root span names instead of all messages? We only query for root spans still, is there a use case where you'd want the metadata in every span name?

Reply:
Added that change now 👍